### ISSUE TITLE
508-defect-3 [COGNITION]: App Directory - Nested lists should have aria-labels to differentiate them

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#16702]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement

As a screen reader user, I want to be able to differentiate the data request lists when I have more than one app accordion expanded.

## Details

JAWS show each of the nested lists as exactly the same thing with no indicator what each list relates to. We could add ARIA labels to these lists and make them more useful to screen reader users in general, and JAWS in particular. Screenshot attached below.

## Acceptance Criteria

- [ ] All nested OL have an `aria-label` attached
- [ ] VSP content team has approved the content for the ARIA labels
- [ ] 0 axe browser plugin violations

## Solution (if known)

```diff
! Check with VSP content for the text in the aria-label attribute

<ol
+ aria-label="Clinical Trial Selector - potential data requests"
  class="vads-u-margin--0 vads-u-margin-top--1 vads-u-padding-left--2p5"
>
  <li>List items</li>
</ul>
```

## WCAG or Vendor Guidance (optional)

* [Info and Relationships: Understanding SC 1.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html

## Screenshots or Trace Logs

![Screenshot (59)](https://user-images.githubusercontent.com/934879/100677668-198cbd00-3331-11eb-8a08-7ed7c19a4a78.png)

**Comment by 1Copenut** [comment]
Retested quickly with JAWS, NVDA, and VoiceOver in production. There was a slight change in IE11 + the Lists menu, but it still reads correctly. All other screen reader and browser pairings worked well and had a much improved experience with `aria-labels` attached. Moving to close.

**Comment by LeKeve** [comment]
@1Copenut 

Looks like this issue was meant to be closed. Please feel free to reopen if anything was missed.
